docs/content/singleton-actors.md

```diff
@@ -0,0 +1,83 @@
+# Singleton Actors
+
+Singleton Actors allow a single source of truth for both operations and data across a multi-server simulation. They are server-side authoritative [Actors](https://docs.unrealengine.com/en-us/Programming/UnrealArchitecture/Actors) that are restricted to one instantiation on SpatialOS. 
+
+There are two kinds of Singleton Actors:
+
+* **Public Singleton Actors** - Singleton Actors which are replicated to clients. `GameState` is a Public Singleton Actor.
+* **Private Singletons** - Singleton Actors which are not accessible to clients. `GameMode` is a Private Singleton Actor.
+
+You can define any class as a Singleton Actor. At the moment the Unreal GDK only supports Public Singleton Actors.
+
+Each server should instantiate their own local version of each Singleton Actor. For `GameMode` and `GameState`, Unreal Engine does this automatically.
+
+Due to Unreal servers spawning their own instances of each Singleton Actor, proper replication and authority management of Singleton Actors becomes a bit tricky. To solve this issue, we have introduced the concept of a Global State Manager (GSM) to enable proper replication of Singleton Actors. The GSM solves the problem of replicating Singleton Actors by only allowing the server with [authority](https://docs.improbable.io/reference/13.1/shared/glossary#read-and-write-access-authority) over the GSM to execute the initial replication of these Actors. All other servers will then link their local Singleton Actors to their respective SpatialOS entity.
```

Comment:
It would be super handy to be able to link to information about the GSM with a subtitle here.

Reply:
This is the doc where the GSM is talked about for now. Since we haven't fully documented how the GDK works, I've made the actual details of how this all works somewhat sparse.